app/views/govuk_publishing_components/component_guide/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render 'govuk_publishing_components/components/title', title: GovukPublishingComponents::Config.component_guide_title %>
+<%= render 'govuk_publishing_components/components/title', title: "Component Guides" %>
```

Comment:
Did you mean this to be Component Guide - without the s?

Reply:
How comes, the title doesn't have the correct value in it?

Reply:
Right, fixed it now.

@DilwoarH: well, technically the title (as in the `<title>` tag) has the correct value - which is the one set in config of the app, but the heading (`<h1>`) of the page should always be "Component Guide".

Reply:
Thanks for explaining Alex, I also noticed that in the first example, the breadcrumbs is `content publisher` then on the second it's `Components guide` but I only see a change to the heading title component. How has this change affected the breadcrumbs?

Reply:
@DilwoarH you're right, I didn't commit the breadcrumbs replacement. Added now, good spot.